source/content/glossar/bibliothek/dcat-definitions/distribution-title.rst

```diff
@@ -1,13 +1,10 @@
 :DCAT URI: dct:title
 :Domain: dcat:Distribution
 :Value: ``rdfs:Literal`` http://www.w3.org/TR/rdf-schema/#ch_literal
-:Mandatory: no - except if the distribution does not contain
-            all the content of the dataset.
-:Cardinality: 0..n (one for each language)
-:Attributes: - Name: ``xml:lang``
-             - Content: ``en``, ``de``, ``fr``, ``it``
-             - Description: Language of the element
-             - Mandatory: yes
-:Description: The title of the distribution in the language defined
-              by the ``xml:lang?`` attribute. If this element is left out,
-              the ``dct:title`` of the dataset is used instead.
+:Requirement Level: mandatory
+:Cardinality: 1..4 (one for each language)
```

Comment:
before it was 0..n and now it is 1..n. I am certain that you know better, what value needs to be here, but maybe double-check that this is not a typo? 😅